.github/workflows/ci.yml

```diff
@@ -27,8 +27,9 @@ jobs:
           go-version-file: go.mod
       - uses: golangci/golangci-lint-action@08e2f20817b15149a52b5b3ebe7de50aff2ba8c5 # v3.4.0
         with:
-          version: v1.50.1
+          version: v1.52.1
```

Comment:
Is it possible to get this version renovated? @mikaelol

Reply:
```suggestion
          # renovate: datasource=go depName=github.com/golangci/golangci-lint
          version: v1.52.1
```